frontend/src/components/GrantAccess.vue

```diff
@@ -1,34 +1,40 @@
 <script setup lang="ts">
-import PageTitle from '@/components/common/PageTitle.vue';
 import { ApiServiceFactory } from '@/services/ApiServiceFactory';
-import { selectedApplication } from '@/store/ApplicationState';
+import {
+    selectedApplication,
+    selectedApplicationDisplayText,
+    grantAccessFormData,
+} from '@/store/ApplicationState';
 import type {
     FamApplicationRole,
     FamForestClient,
     FamUserRoleAssignmentCreate,
 } from 'fam-api';
 import { onMounted, ref } from 'vue';
-import { useToast } from 'vue-toastification';
 import { Form as VeeForm, Field, ErrorMessage } from 'vee-validate';
 import router from '@/router';
 import { number, object, string } from 'yup';
+import Button from 'primevue/button';
+import InputText from 'primevue/inputtext';
+import Dropdown, { type DropdownChangeEvent } from 'primevue/dropdown';
+import RadioButton from 'primevue/radiobutton';
 
 const FOREST_CLIENT_INPUT_MAX_LENGTH = 8;
 const domainOptions = { IDIR: 'I', BCEID: 'B' }; // TODO, load it from backend when backend has the endpoint.
 let applicationRoleOptions = ref<FamApplicationRole[]>([]);
 let forestClient: FamForestClient[] | null = null;
 let loading = ref<boolean>(false);
+let userLoaded = ref<boolean>(false);
```

Comment:
do we need this variable? or we can just check if formData.userId has a value to replace this variable?